docs/bis/ec/index.md

```diff
@@ -1,18 +1,18 @@
 # ECObjects Information Modeling Language Overview
 
-The ECObjects language is comprised of items that have attributes. In this section, we will briefly describe the "ECSchema" element and the kinds of items that are the direct members of an ECSchema element. For each of these kinds of items, we will define their attributes (and other kinds of items that they contain) in subsequent sections.
+The ECObjects language is comprised of items that have attributes. Items are grouped into "domains" via an "ECSchema", which provides the container and namespace for the items it that domain. There are several types of items in an ECSchema that describe objects, relationships, value types and quantities.  Each item type is described below in more detail.
 
-The first kind of element is an *ECSchema* itself.
+The root of any domain is an *ECSchema* itself.
 
-- An **[ECSchema](./ec-schema.md)** describes one "domain"—its primary contents are a closely related set of "ECClasses" of various kinds. The name attribute of the ECSchema serves as a namespace for all elements defined within it. ECSchemas cannot contain other ECSchemas.
+- An **[ECSchema](./ec-schema.md)** describes one "domain"—its primary contents are a closely related set of "ECClasses" of various kinds. The name attribute of the ECSchema serves as a namespace for all items defined within it. All items within an ECSchema have a unique name.
 
-First among the kinds of elements that can be a direct member of an ECSchema is the ECSchemaReference:
+ECSchemas cannot contain other ECSchemas but they can reference other schemas:

```

Comment:
You switch from using ECSchemas consistently to "schemas" at the end here... I'd probably use ECSchemas to be consistent.